packages/templates/src/test/raw.js

```diff
@@ -83,4 +83,20 @@ describe('raw template files', () => {
       }
     }
   );
+
+  // @see https://github.com/google/web-stories-wp/pull/5889
+  it.each(templates)(
+    '%s template should not contain extraneaous properties',
```

Comment:
this is so dope dude

Reply:
Thanks I copy and pasted it 😅

Reply:
speaking of which, I need to update the comment above